### PR TITLE
Added Jada Blankenship's website

### PR DESCRIPTION
**Pull request by jada4569**
Yay I made a website

### CODE REVIEW

**Review by livartly — DISMISSED** [review]
Fantastic work!

**Review by Duke02 — DISMISSED** [review]

**Review by ErikFailing — DISMISSED** [review]

File: members/nathansolomon/index.html [hunk 1]
```diff
@@ -11,5 +11,6 @@ <h1>Welcome to my super basic introductory site!</h1>
 			<p>Have a fun picture of the number 3<p>
 			<img src="images/003.png" alt="003"></img>
 		</div>
+	<p> hi</p>
```
Comment (willBoyd8) [inline members/nathansolomon/index.html]: Changes to other member's websites need to be removed
```suggestion
```

**Review by willBoyd8 — CHANGES_REQUESTED** [review]
Looks great! The changes to @Jedimaster4559's site need to be removed, though

**Review by Jedimaster4559 — CHANGES_REQUESTED** [review]
Take a look at the changes requested by Will. Once you fix those I can approve this!

**Review by Jedimaster4559 — DISMISSED** [review]

File: members/nathansolomon/index.html (hunk 1 above)
Comment (willBoyd8) [inline members/nathansolomon/index.html]:
```suggestion
```

**Review by willBoyd8 — CHANGES_REQUESTED** [review]

**Review by willBoyd8 — APPROVED** [review]

**Review by Jedimaster4559 — APPROVED** [review]

**Review by ErikFailing — APPROVED** [review]

**Pull request merged**